client/ayon_core/hosts/maya/plugins/publish/extract_obj.py

```diff
@@ -54,12 +68,13 @@ def process(self, instance):
                                        pointsShaded=1,
                                        polygonObject=1):
                 with lib.shader(members,
-                                shadingEngine="initialShadingGroup"):
+                                shadingEngine="initialShadingGroup") if strip_shader else nullcontext():  # noqa: E501
```

Comment:
If strip shader now forces `materials=0` on the export. Do we still need to force the initial shading group? Or does that help avoiding other stuff to be exported?

Reply:
to be honest, options on OBJExport are so undocumented that I left that here just to be safe. But I guess it is slowing extraction down unnecessarily.